recipe/meta.yaml

```
@@ -1,16 +1,16 @@
-{% set version = "0.20.1" %}
+{% set version = "0.26.1" %}
 
 package:
   name: modin-packages
   version: {{ version }}
 
 source:
   url: https://github.com/modin-project/modin/archive/refs/tags/{{ version }}.tar.gz
-  sha256: 6d5c3d0fd0ec370af7c8eeb3f1729765aee97d19d3ebaa79f70b9d94a061bae9
+  sha256: acf18f492fd1b7f4fe59aee3a50310650adc6696586a70c0333ac6c160002407
 
 build:
   number: 0
-  skip: true  # [py<38]
+  skip: true  # [py<39 or py>311]
```

```suggestion
  skip: true  # [py<39]
```
In case 3.12 was skipped because of missing ray only.